pimd/pim6_mld.c

```diff
@@ -2847,6 +2848,123 @@ DEFPY(gm_show_interface_joins,
 	return vty_json(vty, js);
 }
 
+static void gm_show_groups(struct vty *vty, struct vrf *vrf, bool uj)
+{
+	struct interface *ifp;
+	struct ttable *tt;
+	char *table;
+	json_object *json = NULL;
+	json_object *json_iface = NULL;
+	json_object *json_group = NULL;
+	json_object *json_groups = NULL;
+	struct pim_instance *pim = vrf->info;
+
+	/* Prepare table. */
+	tt = ttable_new(&ttable_styles[TTSTYLE_BLANK]);
```

Comment:
Btw, do we need to do this stuff even if we don't care about JSON?

Reply:
Will fix this, Thanks